lnrpc/rpc.proto

```diff
@@ -268,10 +268,14 @@ message ActiveChannel {
     int64 total_satoshis_received = 10 [ json_name = "total_satoshis_received" ];
     uint64 num_updates = 11 [ json_name = "num_updates" ];
 
-    repeated HTLC pending_htlcs = 12 [ json_name = "pending_htlcs" ];
+    ChannelStatus status = 12 [ json_name = "status" ];
+
+    repeated HTLC pending_htlcs = 13 [ json_name = "pending_htlcs" ];
 }
 
-message ListChannelsRequest {}
+message ListChannelsRequest {
+    ChannelStatus status = 1;
```

Comment:
Similar comment here as to the one above: this particular diff line is unneeded as the `PendingChanenlsRequest` already has the necessary fields needed.